content/custom-commands/database.md

```diff
@@ -0,0 +1,80 @@
++++
+title = 'Database'
+weight = 2
++++
+
+The Custom Command Database is used for persistent storage between custom command executions. The database page displays
+all database entries created by custom commands, allowing you to view details on or delete individual entries.
+
+<!--more-->
+
+![Overview of the Database page.](overview_database.png)
+
+<center>
+
+**1** Entry ID **2** Created Timestamp **3** Updated Timestamp **4** Expiry Timestamp **5** User ID **6** Key **7**
+Value **8** Size of Value **9** Delete Entry **10** Page Navigation **11** Search Query **12** Search Bar
+
+</center>
+
+## Entries
+
+### Entry ID
+
+The ID is assigned by the system and unique within YAGPDB's database. The database page is sorted by descending ID. You
+may seach for a specific ID using the **Search Bar** (**11**) with ID **Query** (**10**).
+
+### Created Timestamp
+
+UTC timestamp of when the entry was created.
+
+### Updated Timestamp
+
+UTC timestamp of when the entry was updated. Increasing or setting a database value updates this timestamp.
+
+### Expiry Timestamp
+
+UTC timestamp of when the entry will expire. An expired entry will not be retrieved by database functions, but **will**
+appear on the database page.
+
+### User ID
+
+The user-defined ID of the entry (does not have to be a user's ID, accepts any int64). You may search for a specific ID
+using the **Search Bar** (**11**) with User ID **Query** (**10**).
+
+### Key
+
+The user-defined key of the entry. A key is a string, max 256 characters. You may search for a specific key using the
+**Search Bar** (**11**) with Key **Query** (**10**).
+
+### Value
+
+The serialized value of the database entry.
+
+### Size of Value
+
+Size of the value in bytes. YAGPDB database entries have a max value size of 100kB.
```

Comment:
You generally have a (non-breaking) space between the numerical value
and the unit symbol, minus a very few exceptions like `°C`. My
professors also always chastize me when I do not do that, so I feel your
pain. :-)
```suggestion
Size of the value in bytes. YAGPDB database entries have a max value size of 100 kB.
```